tests/foreman/cli/test_contenthost.py

```diff
@@ -588,16 +532,16 @@ def test_contenthost_package_upgrade(self):
         @Assert: Package was successfully upgraded
 
         """
-        self.vm.run('yum install -y {}'.format(FAKE_1_CUSTOM_PACKAGE))
+        self.client.run('yum install -y {}'.format(FAKE_1_CUSTOM_PACKAGE))
```

Comment:
Addressed and all instances of this is fixed in https://github.com/SatelliteQE/robottelo/pull/2855